rocky/reports/views/base.py

```diff
@@ -267,12 +267,30 @@ def is_scheduled_report(self) -> bool:
         recurrence_choice = self.request.POST.get("choose_recurrence", "once")
         return recurrence_choice == "repeat"
 
-    def create_report_recipe(self, report_name_format: str, subreport_name_format: str, schedule: str) -> ReportRecipe:
+    def get_object_selection(self) -> str:
+        return self.request.POST.get("object_selection", "")
```

Comment:
This function is only referenced and called once, so you might as well just inline the code instead of creating this function. Also, if it was referenced multiple times, a property would be more appropriate than a getter function